Breeze.UI/tsconfig.json

```diff
@@ -17,7 +17,7 @@
     },
     "types": [
       "node",
-      "jasmine"
+      "jasmine",
```

Comment:
No need for a comma

Reply:
Not something I touched.  Have changed all the same.